go/client/cmd_chat_api.go

```diff
@@ -239,7 +223,7 @@ func (c *CmdChatAPI) ReadV1(ctx context.Context, opts readOptionsV1) Reply {
 	if opts.ConversationID == 0 {
 		// resolve conversation id
 		query := c.getInboxLocalQuery(opts.ConversationID, opts.Channel)
-		gilres, err := client.GetInboxLocal(ctx, chat1.GetInboxLocalArg{
+		gilres, err := client.GetInboxAndUnboxLocal(ctx, chat1.GetInboxAndUnboxLocalArg{
```

Comment:
Yes I think you are right; My concern here was about getting conversation using ID, but that's the case anyway, so we should rely on verifying TLF name afterwards I guess